chaincode/compute_plan.go

```diff
@@ -220,6 +224,7 @@ func createComputePlanInternal(db *LedgerDB, inp inputComputePlan) (resp outputC
 		resp.TesttupleKeys = append(resp.TesttupleKeys, testtupleKey)
```

Comment:
We should add the testtuples' keys here I think.

Reply:
After a quick exchange with @inalgnu , in the case of the `createComputePlan`, all the testtuples are supposed to be in a `waitingStatus` so there are not relevant to determine the compute plan global status. 
Sorry for the noise.